addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -821,6 +830,16 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         // We need to reload the existing type list.
         this.rerender = true;
     },
+    /**
+     * Rerender the clone to avoid id duplicates.
+     *
+     * @override
+     */
+    async onClone() {
+        this._renderList();
+        const field = this._getActiveField();
+        await this._replaceField(field);
+    },
```

Comment:
`onClone` cannot be async, PR to review. If really no other way we could change the API to allow it, but not entirely stable...

Reply:
@qsm-odoo Removing the `async` `await` part of this diff seems fine functionally? Is there any issue doing that?

Reply:
If `_replaceField` does async stuff, it must not be ignored. One example of what could go wrong:

1. Start a history step for the editor
2. Clone (not awaiting _replaceField)
3. End history step for the editor
4. _replaceField modifies stuff related to the async stuff which now finishes => your history step is wrong

That may not be the case here, I don't know. But it is very rarely (if ever) a good idea to ignore async parts, that's the best way to have race condition everywhere.

Reply:
The most important part here is to call `_renderField` AFAIK, as this is the code that is handling IDs (the most important part being the "same ID + increment suffit" for multiple checkbox/select)
@younn-o is there any chance we can call `_renderField` (+ some extra code ofc) instead of `_replaceField` ?

Reply:
I think that we can skip fetching the field records in the replaceField method, which is the async part. I think it's not mandatory when cloning because the fields have already been fetched for the original field to clone. I added _replaceHtmlField which is synchronous and can be called from onClone